Content/NPCs/Dungeon/CrescentCaster.cs

```diff
@@ -0,0 +1,505 @@
+﻿//TODO:
+//[3]Bestiary
+
+
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using ReLogic.Content;
+using StarlightRiver.Content.Abilities;
+using StarlightRiver.Content.Dusts;
+using StarlightRiver.Content.Abilities.ForbiddenWinds;
+using StarlightRiver.Content.Items.Misc;
+using StarlightRiver.Core;
+using StarlightRiver.Helpers;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.Graphics.Effects;
+using System.Collections.Generic;
+
+using Terraria.DataStructures;
+using Terraria.GameContent;
+
+using Terraria.Audio;
+
+using System;
+using System.Linq;
+using Terraria.GameContent.Bestiary;
+using Terraria.GameContent.ItemDropRules;
+using static Terraria.ModLoader.ModContent;
+using Terraria.GameContent.Bestiary;
+using Terraria.ModLoader.Utilities;
+
+
+namespace StarlightRiver.Content.NPCs.Dungeon
+{
+    public class CrescentCasterBolt
```

Comment:
This feels like it should be placed at the bottom of the file as opposed to the top, or a seperate file altogether. In general lets try to keep the top class of a file the namesake of the file.

Reply:
fixed